DBMS/DBMSAssignment4/dbmsassignment4.sql

```diff
@@ -0,0 +1,55 @@
+  -- Assignment 1
+USE lib;
+ -- qn 1
+
+SELECT member_name FROM members  WHERE category IN(SELECT category FROM members WHERE member_name="keshav sharma");
+-- qn 2
+SELECT bi.issue_date,t.title_name,m.member_name,bi.due_date FROM book_issue bi,title t,members m ,books b 
+WHERE bi.accession_no=b.accession_no 
+AND b.title_id=t.title_id
+AND bi.member_id=m.member_id
+AND (bi.member_id,bi.accession_no) NOT IN(SELECT br.member_id,br.accession_no FROM book_return br);
+
+-- qn 3
+SELECT bi.issue_date, t.title_name, m.member_name ,bi.due_date FROM book_issue bi,title t,members m ,books b
+WHERE bi.accession_no=b.accession_no 
+AND b.title_id=t.title_id
+AND bi.member_id=m.member_id
+AND exists(SELECT  br.member_id,br.accession_no FROM book_return br WHERE datediff(bi.due_date,br.return_date)<0);
+
+-- qn 4 
+SELECT * FROM books WHERE price = (SELECT max(price) FROM books);
+
+  -- qn 5
+SELECT price FROM(SELECT price FROM books 
```

Comment:
It could have been better done with the following query:-

select max(price) from books where price <> (select max(price) from books)